workbench/settings.py

```diff
@@ -97,13 +116,6 @@
 # Make this unique, and don't share it with anybody.
 SECRET_KEY = '5ftdd9(@p)tg&amp;bqv$(^d!63psz9+g+_i5om_e%!32%po2_+%l7'
 
-# List of callables that know how to import templates from various sources.
-TEMPLATE_LOADERS = (
```

Comment:
I had not read that documentation! If we're using the default values, then we shouldn't specify them at all. 👍 
I'm going to make an edx-platform PR to do the same...